### PR TITLE
Fix Typos in the Codebase

### PR DESCRIPTION
**Pull request by faze-geek**
I just cloned `pocketpy` on my windows machine.  I am using the MVSC compiler and could successfully build `pocketpy`.
I made this PR just to test the initial setup by running `codespell` over the `pocketpy` codebase.

### CODE REVIEW

**Comment by faze-geek** [comment]
@blueloveTH I see there is no automated testing through CI here, hopefully contributors are expected to push code here itself.

**Review by blueloveTH — CHANGES_REQUESTED** [review]
Hello @faze-geek 
I've made some comments in your changes!

File: 3rd/cjson/LICENSE
Comment (blueloveTH) [inline 3rd/cjson/LICENSE:1]: Don't change 3rd files.

File: benchmarks/res/WorldMap_Free_layout.ldtk
Comment (blueloveTH) [inline benchmarks/res/WorldMap_Free_layout.ldtk:1]: Don't change 3rd files.

File: c_bindings/test.c
Comment (blueloveTH) [inline c_bindings/test.c:1]: This could cause error.

File: 3rd/cjson/LICENSE
Comment (faze-geek) [reply 3rd/cjson/LICENSE:1]: Reverted

File: c_bindings/test.c
Comment (faze-geek) [reply c_bindings/test.c:1]: Reverted

**Comment by faze-geek** [comment]
> I've made some comments in your changes!

I have made the changes. Thanks for pointing them out, `codespell` does tend to give false positives sometimes.

File: benchmarks/res/WorldMap_Free_layout.ldtk
Comment (blueloveTH) [reply benchmarks/res/WorldMap_Free_layout.ldtk:1]: I know. This file was copied from LDtk, not written by us. There is no need to fix it.

Comment (faze-geek) [reply benchmarks/res/WorldMap_Free_layout.ldtk:1]: Sure. I'll make the changes quick.

**Comment by faze-geek** [comment]
Made all the suggested changes :)

**Review by blueloveTH — APPROVED** [review]

**Comment by blueloveTH** [comment]
Thank you for your correction ~

**Pull request merged**